builds/e2e/stresstest.yaml

```diff
@@ -358,4 +358,67 @@ jobs:
           snitch.alert.url: '$(SnitchStressAlertUrl)'
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
+          loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
+
+################################################################################
+  - job: windows_servercore_amd64_moby
+################################################################################
+    displayName: Windows Server Core AMD64 Moby
+    condition: eq(variables['run.windows.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - Agent.OSName -equals ServerCore
+        - run-stress -equals true
+    variables:
+      edgelet.artifact.name: 'iotedged-windows'
+    steps:
+      - checkout: none
+      - task: AzureKeyVault@1
+        displayName: 'Azure Key Vault'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: $(azure.keyVault)
+          SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,IotHubStressConnString,EventHubStressConnStr,StorageAccountMasterKeyStress,SnitchStressAlertUrl'
+      - task: DownloadBuildArtifacts@0
```

Comment:
Create template to include these 2 download artifact tasks for reuse same as the above comment for long haul.

Reply:
As mentioned in the other comment, going to leave the template work for a later date since that's more refactoring work. I attempted this to see if I could do it quickly, and was running into some issues with running it. It needs more time to iron out the issues.

Additionally, if we do this later, then we can also change the IoT Core stress and long-haul configs as well.

Reply:
I think we did spot out the issue is about same value is passed to template for pipeline parameter; which there are 2 different pipeline for download edgelet and images artifacts.  Please help to continue refactoring this after this PR is committed.

Reply:
Yes, adding the template should be pretty straight forward. This task has been added to our backlog, which will be prioritized based on other work that we have.